src/shared/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -87,6 +88,31 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
 
   const displayPlayOverlay = playOverlayVisible && !isInBackground
 
+  // handle hotkeys
+  useEffect(() => {
+    if (!player || isInBackground) {
+      return
+    }
+
+    const handler = (e: KeyboardEvent) => {
```

Comment:
Meaningful names please 🙂
```suggestion
    const handler = (event: KeyboardEvent) => {
```